frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_detail/mobile_card_detail_screen.dart

```diff
@@ -135,49 +135,44 @@ class _MobileRowDetailPageState extends State<MobileRowDetailPage> {
       builder: (_) => Column(
         mainAxisSize: MainAxisSize.min,
         children: [
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 8.0),
-            child: MobileQuickActionButton(
-              onTap: () {
-                final rowId = _bloc.state.currentRowId;
-                if (rowId == null) {
-                  return;
-                }
-                RowBackendService.duplicateRow(viewId, rowId);
-                context
-                  ..pop()
-                  ..pop();
-                Fluttertoast.showToast(
-                  msg: LocaleKeys.board_cardDuplicated.tr(),
-                  gravity: ToastGravity.BOTTOM,
-                );
-              },
-              icon: FlowySvgs.copy_s,
-              text: LocaleKeys.button_duplicate.tr(),
-            ),
+          MobileQuickActionButton(
+            onTap: () {
+              final rowId = _bloc.state.currentRowId;
+              if (rowId == null) {
+                return;
+              }
+              RowBackendService.duplicateRow(viewId, rowId);
+              context
+                ..pop()
+                ..pop();
+              Fluttertoast.showToast(
+                msg: LocaleKeys.board_cardDuplicated.tr(),
+                gravity: ToastGravity.BOTTOM,
+              );
+            },
+            icon: FlowySvgs.copy_s,
+            text: LocaleKeys.button_duplicate.tr(),
           ),
-          const Divider(height: 9),
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 8.0),
-            child: MobileQuickActionButton(
-              onTap: () {
-                final rowId = _bloc.state.currentRowId;
-                if (rowId == null) {
-                  return;
-                }
-                RowBackendService.deleteRow(viewId, rowId);
-                context
-                  ..pop()
-                  ..pop();
-                Fluttertoast.showToast(
-                  msg: LocaleKeys.board_cardDeleted.tr(),
-                  gravity: ToastGravity.BOTTOM,
-                );
-              },
-              icon: FlowySvgs.m_delete_m,
-              text: LocaleKeys.button_delete.tr(),
-              color: Theme.of(context).colorScheme.error,
-            ),
+          const Divider(height: 8.5, thickness: 0.5),
+          MobileQuickActionButton(
+            onTap: () {
+              final rowId = _bloc.state.currentRowId;
+              if (rowId == null) {
+                return;
+              }
+              RowBackendService.deleteRow(viewId, rowId);
+              context
+                ..pop()
+                ..pop();
+              Fluttertoast.showToast(
+                msg: LocaleKeys.board_cardDeleted.tr(),
+                gravity: ToastGravity.BOTTOM,
+              );
+            },
```

Comment:
How about unifying the two onTaps? They do mostly the same except a different action.
```
void _performAction(String viewId, String? rowId, bool deleteRow) {
  if (rowId == null) {
    return;
  }

  // If not deletion, then it's duplication
  deleteRow
    ? RowBackendService.deleteRow(viewId, rowId)
    : RowBackendService.duplicateRow(viewId, rowId);

  context
    ..pop()
    ..pop();
  Fluttertoast.showToast(
    msg: LocaleKeys.board_cardDeleted.tr(),
    gravity: ToastGravity.BOTTOM,
  );
}
```

```suggestion
            onTap: () => 
              _performAction(viewId, _bloc.state.currentRowId, true),
```